docs/cep/table/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Table Stores",
```

Comment:
@diana-macrometa - Why `Table Stores`, I think only `Table(s)` is enough?

Reply:
I do not recall, but I can change it back.